index_impl.go

```diff
@@ -368,6 +368,25 @@ func (i *indexImpl) Search(req *SearchRequest) (sr *SearchResult, err error) {
 	return i.SearchInContext(context.Background(), req)
 }
 
+var documentMatchSizeEstimate int
+var searchContextSizeEstimate int
+var facetResultSizeEstimate int
+var documentSizeEstimate int
```

Comment:
minor thing on naming -- these don't feel like estimates to me, but seem pretty deterministic

Reply:
that is, if you remove the suffix "Estimate", then it'd have more correct naming?

Reply:
or, perhaps replace "SizeEstimate" with "EmptySize"?

Reply:
Sounds good.